### ISSUE TITLE
APK download page: Generic test QR codes uncalled-for

### ISSUE THREAD

**Issue by lasermind** [Peach2Peach/public-website#173]
There seems to be a problem with the download page: there are two QR codes displayed, leading practically to nowhere. 🤔

![qrcodesonapkpage](https://github.com/user-attachments/assets/59ae69f9-15c3-4da0-a8ad-1c4f8bbcd803)

It's the APK download page, there should not be a link to the Play Store or the App Store.

**Comment by HellGio** [comment]
Hi! I confirm that it's solved now

**Comment by lasermind** [comment]
Yes, the QR codes seem to work now, but the page is still a bit messed up. A reordering of the elements would make more sense.
